src/WebUI/Pods/PodsLeftPanel.tsx

```diff
@@ -48,15 +51,25 @@ export class PodsLeftPanel extends BaseComponent<IPodsLeftPanelProperties> {
         );
     }
 
+    public componentDidMount() {
+        // Select the first pod in left panel by default
+        this._selection.select(0);
```

Comment:
should it be good to have a check whether the count is more than zero and then select zero or we shoudl be good? also when the same UI is shown for someother pod, do we select that needed POD?

Reply:
1. We are defining this._selection in the constructor itself, the Selection object itself takes care of all cases. https://dev.azure.com/mseng/AzureDevOps/_git/AzureDevOps?path=%2FVssf%2FWeb%2Fextensions%2Fvss-features%2Fvss-ui%2Fvss-ui-core%2FUtilities%2FSelection.ts&version=GBmaster&line=162&lineStyle=plain&lineEnd=162&lineStartColumn=5&lineEndColumn=66
2. Yes the selection is highlighted in the left panel according to the selected pod